.github/workflows/on_pull_request.yml

```diff
@@ -401,18 +471,26 @@ jobs:
       - name: Install Dependencies
         run: yarn install --immutable
 
-      - name: Download build artifacts
+      - name: Download website build artifact
         uses: actions/download-artifact@v2
         with:
-          name: compiled-js-and-types
-          path: packages/
+          name: compiled-website
+          path: packages/paste-website/
+          if-no-files-found: error
```

Comment:
ℹ️ I noticed that if the save-cache step _fails_ it just throws a warning, and does not error out. This will cause the job to fail and the action to stop running if no files are found on download.